client/src/pages/locations/Form.js

```diff
@@ -56,16 +56,25 @@ const LOCATION_TYPES_ADMIN = [
 const LOCATION_TYPES_SUPERUSER =
   Settings?.fields?.location?.superuserTypeOptions
 
-const LocationForm = ({ edit, title, initialValues, notesComponent }) => {
+const LocationForm = ({
+  edit,
+  title,
+  initialValues,
+  notesComponent,
+  afterSaveActions,
+  afterCancelActions
+}) => {
   const { currentUser } = useContext(AppContext)
   const navigate = useNavigate()
   const [error, setError] = useState(null)
   const [attachmentList, setAttachmentList] = useState(
     initialValues?.attachments
   )
   const [showSimilarLocations, setShowSimilarLocations] = useState(false)
+  const regularUsersCanCreateLocations = Settings.regularUsersCanCreateLocations
```

Comment:
Note that regular users can't create a location as the changes are incomplete: the location type is required but regular users can't select a location type in the dropdown. Additional changes to this Form are needed!

Reply:
:( I was sure I had tested this...